tests/main/test_task.py

```diff
@@ -673,10 +676,14 @@ def polinsky_feedback(self, msg: FeedbackTool) -> str | DonePassTool:
     assert all(s in response.content for s in strings)
 
 
-def test_task_tool_responses():
+def test_task_tool_responses(
+    test_settings: Settings,
```

Comment:
don't need this any more since `conftest.py` now has a fixture that auto-injects these